5_databases/week1/HW.sql

```diff
@@ -0,0 +1,62 @@
+-- Active: 1699188605886@@0.0.0.0@3306@hyf_lesson1
+
+
+-- Task 1 Find out how many tasks are in the task table
+SELECT COUNT (title)
```

Comment:
Not a valid syntax, remove space between COUNT and `(title)`. Use `(*)` instead as we dont need to specify field